### ISSUE TITLE
Hi, is version 3.0 safe to use in production ? Thanks

### ISSUE THREAD

**Issue by shannoga** [magicalpanda/MagicalRecord#1118]

**Comment by tonyarnold** [comment]
Yep, should be. I think both @casademora and I have shipped apps with version 3.0 in them — we just haven't tidied everything up into a releasable state yet.

**Comment by hardikdevios** [comment]
@tonyarnold thanks

**Comment by pronebird** [comment]
@tonyarnold can you make a release of 3.0? thanks.

**Comment by domhof** [comment]
Any updates on when 3.0 will be released?

**Comment by AnthonyMyatt** [comment]
I too would like to know when 3.0 will be released. I am using it in a project, however due to the lack of proper documentation I am not sure if I am doing everything correctly.

**Comment by tonyarnold** [comment]
I don't think anyone is working on MagicalRecord regularly anymore. I haven't had time for months, and nobody else is contributing patches.

Unless someone is able to jump in and help out, this is where things are for the foreseeable future.
